libr/asm/asm.c

```diff
@@ -316,9 +317,10 @@ R_API bool r_asm_use_assembler(RAsm *a, const char *name) {
 R_API int r_asm_use(RAsm *a, const char *name) {
 	RAsmPlugin *h;
 	RListIter *iter;
-	if (!a || !name) {
-		return false;
-	}
+
+	r_return_val_if_fail (a, false);
```

Comment:
asm_use should return bool

Reply:
That's not realted to this pr I believe.